### ISSUE TITLE
Infinite Recursion in VerticaConfiguration constructor

### ISSUE THREAD

**Issue by dquigleyzynga** [vertica/Vertica-Hadoop-Connector#6]
https://github.com/vertica/Vertica-Hadoop-Connector/blob/master/hadoop-connector/com/vertica/hadoop/VerticaConfiguration.java#L209

**Comment by billonahill** [comment]
Yup, looks like level should be removed from call in line 209. Same below for logPath on line 237. Since these constructors are clearly not in use they could also be removed.
